server/lib/ckb_authentic_sync.rb

```diff
@@ -1,12 +1,9 @@
-require "daemons"
 require_relative "../config/environment"
 
-options = {
-  log_output: true,
-  log_dir: Rails.root.join("log"),
-  monitor: true,
-  dir: Rails.root.join("tmp", "pids").to_s
-}
-Daemons.run_proc("#{Rails.env}_ckb_authentic_sync", options) do
-  CkbSync::AuthenticSync.start
-end
+loop do
```

Comment:
Why not use rake tasks

Reply:
After using `system to  start daemons, I don't think it is necessary to use rake tasks.

Reply:
rake task is a process in system, `systemd` to manage this process, this logic looks like a task and not belongs to main process.